Calling/Controllers/CallRecordingController.cs

```diff
@@ -343,11 +344,12 @@ public async Task<ActionResult> GetRecordingFile([FromBody] object request)
 
                     if(!string.IsNullOrWhiteSpace(cloudEvent.Subject))
                     {
-                        string[] subjectPaths = cloudEvent.Subject.Split("/");
+                        var recIdMatch = Regex.Match(cloudEvent.Subject, @"(recordingId/[^/]*)");
```

Comment:
Could you share a sample string of what syntax we are getting from cloudEvent.Subject? Is it something we can define in the cloudEvent, wondering if that is possible to put recordingId into cloudEvent.Data object (try to avoid regex as much as we could)

but yea, for this PR itself, it looks good

Reply:
Here is the format of the subject. /recording/call/<call-id>/serverCallId/<server-call-id>/recordingId/<recording-id>